src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -293,5 +294,7 @@
     </None>
   </ItemGroup>
 
-
+  <ItemGroup>
+    <PackageReference Include="Microsoft.IdentityModel.Abstractions" Version="6.18.0" />
```

Comment:
6.18 ? What is the versioning story for `Microsoft.IdentityModel.Abstractions` ?

Reply:
Not sure, Ill ask Sruthi